src/main/java/uk/gov/pay/adminusers/resources/ToolboxEndpointResource.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.pay.adminusers.resources;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.adminusers.service.ServiceServicesFactory;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.Status.NO_CONTENT;
+
+@Path("/v1/api/toolbox")
```

Comment:
Having this explicitly behind a `/toolbox` or `/admin` route feels like a good pattern to avoid other consumers accidentally using this 👍